docu/files/ModuleExtensibility.html.md

```diff
@@ -114,12 +110,30 @@ BuilderExtensible.on('myPluginEvent', function(data) {
 
 ### Register host and plug-in modules
 
-Extensible hosts and plug-in modules, just like all other Builder modules, must be registered in YaaS.
-So, after you have implemented and deployed them, use the Builder to register the modules in YaaS and include them in the target packages that your project provides.   
+Extensible hosts and plug-in modules must be registered in YaaS just like all other Builder modules. So, after you have implemented and deployed them, use the Builder to register the modules in YaaS. Include the hosts and plug-in modules in the target packages that your project provides.   
```

Comment:
Yuou must register extensible hosts and plug-in modules in YaaS just like all other Builder modules. So, after you implement and deploy your them, use the Builder to register the modules in YaaS. Include the hosts and plug-in modules in the target packages that your project provides.